simple-promise-queue/src/index.spec.js

```diff
@@ -62,13 +62,13 @@ test('handles concurrency of promises', async () => {
   promises = [];
   target = [];
 
+  const t = Date.now();
   const qDoSth = queueable(doSth, {  concurrency: 5 });
 
   for (let i = 0; i < 10; i++) {
     promises.push(qDoSth());
   }
 
-  const t = Date.now();
```

Comment:
@flipace like this?

Reply:
should work better, yeah, i think it should come right after `const qDoSth`, but in any way it may still yield somewhat unexpected results...javascript and timers are not really precise